### ISSUE TITLE
parsing error for immediate-mode (backquote) expressions containing quoted equals ("=")

### ISSUE THREAD

**Issue by karlmsmith** [NOAA-PMEL/Ferret#962]
**Reported by andrew wittenberg on 19 Sep 2009 21:42 UTC**
Apparently immediate-mode (backquotes) thinks any "=" inside a quoted string indicates an immediate-mode option, when it should just interpret the string as a string.

```
        NOAA/PMEL TMAP
        FERRET v6.203  
        Linux(g77) 2.4.21-32 - 06/26/09
        19-Sep-09 17:39     

yes? say `"+" EQ " "`
 !-> MESSAGE/CONTINUE 0
0
yes? say `"=" EQ " "`
 **ERROR: command syntax: options: "P=prec","B=bad","W=width","ZW=zero-width" or "R=return-item"
          MESSAGE/CONTINUE `"=" EQ " "`
```

Migrated-From: http://dunkel.pmel.noaa.gov/trac/ferret/ticket/1690

**Comment by karlmsmith** [comment]
**Comment by andrew.wittenberg on 19 Sep 2009 21:43 UTC**
Of course, a workaround is use LET to define a variable first, and then evaluate that variable with the backquotes.

**Comment by karlmsmith** [comment]
**Comment by andrew.wittenberg on 10 Mar 2010 17:25 UTC**
#1690 closed as a duplicate of #1469.
